.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+Thank you for contributing to KLEE.  We are looking forward to reviewing your PR.  However, given the small number of active reviewers and our limited time, it might take a while to do so.  We aim to get back to each PR within one month, and often do so within one week. 
+
+To help expedite the review please ensure the following, by adding an "x" for each completed item:
+
+- [ ] The PR addresses a single issue.  In other words, if some parts of a PR could form another independent PR, you should break this PR into multiple smaller PRs.
+- [ ] There are no unnecessary commits. For instance, commits that fix issues with a previous commit in this PR are unnecessary and should be removed (you can find [documentation on squashing commits here](https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request#squash-your-changes)).
+- [ ] Larger PRs are divided into a logical sequence of commits.
+- [ ] Each commit has a meaningful message documenting what it does.
+- [ ] The code is commented.  In particular, newly added classes and functions should be documented.
+- [ ] The patch is [clang-formated](https://clang.llvm.org/docs/ClangFormat.html).  Please only format the patch itself and code surrounding the patch, not entire files. 
```

Comment:
Maybe put the reference to [git-clang-format](https://raw.githubusercontent.com/llvm/llvm-project/master/clang/tools/clang-format/git-clang-format) as well. And directions to use it are at the beginning of the file.